From 44cdd36855728059e58dcad52b42d825bb1cc2af Mon Sep 17 00:00:00 2001
From: Paul Fertser <fercerpav@gmail.com>
Date: Fri, 25 Sep 2020 18:36:45 +0300
Subject: [PATCH] configs/ast-common: use 57600 baud rate to match Tioga Pass
 EFI firmware

This is a Tioga Pass specific patch, only for
meta-facebook/meta-tiogapass layer.

Signed-off-by: Paul Fertser <fercerpav@gmail.com>
---
 include/configs/ast-common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/ast-common.h b/include/configs/ast-common.h
index a0243083bd..f0f2ae08b6 100644
--- a/include/configs/ast-common.h
+++ b/include/configs/ast-common.h
@@ -56,7 +56,7 @@
 #define CONFIG_SYS_LOADS_BAUD_CHANGE
 #define CONFIG_SERIAL1				1
 #define CONFIG_CONS_INDEX			1
-#define CONFIG_BAUDRATE				115200
+#define CONFIG_BAUDRATE				57600
 
 /*
  * BOOTP options
@@ -96,7 +96,7 @@
 #define CONFIG_CMD_MTDPARTS
 #endif
 
-#define CONFIG_BOOTARGS			"console=ttyS4,115200n8 root=/dev/ram rw"
+#define CONFIG_BOOTARGS			"console=ttyS4,57600n8 root=/dev/ram rw"
 
 #define CONFIG_AST_SPI_NOR    /* AST SPI NOR Flash */
 #define CONFIG_FMC_CS			1
-- 
2.17.1

